README.md

```diff
@@ -4,6 +4,10 @@ The corefx repo contains the library implementation (called "CoreFX") for [.NET
 
 [mscorlib]: https://github.com/dotnet/coreclr/tree/master/src/mscorlib
 
+## Source Index
+
+.NET Core now has a source index: https://source.dot.net
```

Comment:
Can we add something like "This only includes the managed code in the repo and does not include any of the native libraries.  Additional, for libraries that have different implementations between operating systems, it currently only indexes the Windows implementation; support for also indexing implementations for other operating systems is planned."